hack/images

```diff
@@ -30,6 +30,7 @@ imageDocker() {
 		echo "PREFER_BUILDCTL=1 needs to be set to build images for $PLATFORMS"
 		exit 1
 	fi
+	export DOCKER_BUILDKIT=1
 	set -x
 	docker build $progressFlag --platform=$PLATFORMS -t $REPO:$TAG .
 	docker build $progressFlag --platform=$PLATFORMS -t $REPO:$TAG-rootless --target rootless .
```

Comment:
Does the `export` above persist in the current shell after running this script?

Perhaps instead do

```suggestion
	DOCKER_BUILDKIT=1 docker build $progressFlag --platform=$PLATFORMS -t $REPO:$TAG .
	DOCKER_BUILDKIT=1 docker build $progressFlag --platform=$PLATFORMS -t $REPO:$TAG-rootless --target rootless .
```

Reply:
PR updated, @thaJeztah please help to check again~